addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -42,13 +42,15 @@ export class MediaDialog extends Component {
 
         this.rpc = useService('rpc');
         this.orm = useService('orm');
+        this.notificationService = useService('notification');
 
         this.tabs = [];
         this.selectedMedia = useState({});
 
         this.initialIconClasses = [];
 
         this.addTabs();
+        this.errorMessages = [];
```

Comment:
Should not it be an object instead of an array? Thanks you javascript to allow to write anything and it works but... :smile:

Reply:
Indeed :grimacing: